hal/src/nRF52840/device_code.cpp

```diff
@@ -85,7 +84,7 @@ int get_device_name(char* buf, size_t size) {
     if (nameSize == 0 || nameSize > DEVICE_NAME_MAX_SIZE || !isPrintable(name, nameSize)) {
         // Get device setup code
         char code[SETUP_CODE_SIZE] = {};
-        ret = getDeviceSetupCode(code);
+        ret = get_device_setup_code(code, SETUP_CODE_SIZE);
```

Comment:
Nit: prefer `sizeof(code)`. When refactoring it's easy to forget to replace the size in both places: declaration and function call.